src/examples/pagination/OffsetPadding.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Pagination } from '@zendeskgarden/react-pagination';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const StyledCol = styled(Col)`
+  margin-top: ${p => p.theme.space.sm};
+`;
+
+const Example = () => {
+  const [page, setPage] = useState(0);
+
+  return (
+    <Row alignItems="center">
+      <Col alignSelf="center">
+        <Pagination totalPages={20} pagePadding={1} currentPage={page} onChange={setPage} />
+      </Col>
+      <StyledCol alignSelf="center">
+        <Pagination totalPages={20} pagePadding={3} currentPage={page} onChange={setPage} />
```

Comment:
Can something be done about this one on mobile? Maybe `display: none` (but that kind of defeats the purpose of the example).

Reply:
Brought the page padding number down so it doesn't extend so far outside the container. Happy to adjust.